beat/covariance.py

```diff
@@ -305,6 +305,48 @@ def get_data_covariances(
 
         return cov_ds
 
+    def get_spectradata_covariances(
+            self, wmap, sample_rate, results=None, chop_bounds=None,
+            freq_domain=False, pad_to_pow2=True):
+        """
+        Estimated data covariances of seismic traces
+
+        Parameters
+        ----------
+        wmap : :class:`beat.WaveformMapping`
+        results
+        sample_rate : float
+            sampling rate of data_traces and GreensFunction stores
+
+        Returns
+        -------
+        :class:`numpy.ndarray`
+        """
+        if chop_bounds is None:
```

Comment:
put this stuff until line 334 into a get_structure including the freq_domain flag the rest is identical, isnt it?